StacMan/Codegen/StacManClient.SiteMethods.cs

```diff
@@ -32,7 +32,7 @@ Task<StacManResponse<Site>> ISiteMethods.GetAll(string filter, int? page, int? p
             ub.AddParameter("page", page);
             ub.AddParameter("pagesize", pagesize);
 
-            return CreateApiTask<Site>(ub, HttpMethod.GET, "/sites");
+            return await CreateApiTask<Site>(ub, HttpMethod.GET, "/sites");
```

Comment:
Tip: for all of these where you're only awaiting the result and not otherwise impacted (e.g. no `using` or `try/catch`, etc.), there's extra cost in the state machine, so leaving async/await off is an optimization for APIs (you'll see the same throughout StackExchange.Redis as an example)

Reply:
ah, good to know. I'll back out that change to the T4 template.